.golangci.yml

```diff
@@ -59,3 +59,4 @@ issues:
   - "func SetDefaults_SystemPackages should be SetDefaultsSystemPackages"
   - "type name will be used as kubeone.KubeOneCluster by other packages"
   - "G402: TLS InsecureSkipVerify set true"
+  - "methods on the same type should have the same receiver name"
```

Comment:
this will kill it globally, please don't

Reply:
This is now gone since I removed the semver API.